File: Problem 1/C++/prob1.cpp
#include <iostream>

int main()
{
	long long int sum = 0;

	int k = 1000;

	for(int i = 1; i < k; ++i)
	{
		if(i % 3 == 0 || i % 5 == 0)
		{
			sum+=i;
		}
	}

	std::cout << sum << std::endl;
}

File: Problem 1/Python/Prob1.py
def sum(k):
	sum = 0
	for i in range(1, k):
		if (i % 3 == 0) or (i % 5 == 0):
			sum+=i
	return sum

def main():
	x = int(input("What's the number? "))
	print("The answer is: ", sum(x))

if __name__ == "__main__":
    main()